openmc/summary.py

```diff
@@ -477,10 +478,14 @@ def _finalize_geometry(self):
 
             # Retrieve the object corresponding to the fill type and ID
             if fill_type == 'normal':
-                if fill_id > 0:
-                    fill = self.get_material_by_id(fill_id)
+                if isinstance(fill_id, Iterable):
+                    fill = [self.get_material_by_id(mat) if mat > 0 else 'void'
+                            for mat in fill_id]
                 else:
-                    fill = 'void'
+                    if fill_id > 0:
+                        fill = self.get_material_by_id(fill_id)
+                    else:
+                        fill = 'void'
```

Comment:
But then I'd need a line continuation to keep it under 80 cols so I figure we may as well leave the verbose form.